server/jobs/daily_summary_job.go

```diff
@@ -25,16 +26,9 @@ func NewDailySummaryJob() RegisteredJob {
 
 // runDailySummaryJob delivers the daily calendar summary to all users who have their settings configured to receive it now
 func runDailySummaryJob(env mscalendar.Env) {
-	mscal := mscalendar.New(env, "")
-	_, err := mscal.GetRemoteUser(env.BotUserID)
-	if err != nil {
-		env.Logger.Errorf("Please connect bot user using `/%s connect_bot`, in order to run the daily summary job.", config.CommandTrigger)
-		return
-	}
-
 	env.Logger.Debugf("Daily summary job beginning")
 
-	err = mscal.ProcessAllDailySummary()
+	err := mscalendar.New(env, "").ProcessAllDailySummary(time.Now())
```

Comment:
nit: should this be `BotUserID`?

Reply:
@levb Yes it is implicitly `BotUserID`. The `New` function falls back to the `BotUserID` when an empty string is passed in. It's convenient but I think I prefer the explicit way. Leaving it as is, as we have done this elsewhere and we this is already QA reviewed etc